src/MLJInterface.jl

```diff
@@ -389,17 +393,284 @@ MLJModelInterface.metadata_model(
     path="LightGBM.MLJInterface.LGBMClassifier",
     input=MLJModelInterface.Table(MLJModelInterface.Continuous),
     target=AbstractVector{<:MLJModelInterface.Finite},
-    weights=true,
-    descr="Microsoft LightGBM FFI wrapper: Classifier",
+    weights=true
+    # descr="Microsoft LightGBM FFI wrapper: Classifier",
 )
 
 MLJModelInterface.metadata_model(
     LGBMRegressor; # end positional args
     path="LightGBM.MLJInterface.LGBMRegressor",
     input=MLJModelInterface.Table(MLJModelInterface.Continuous),
     target=AbstractVector{MLJModelInterface.Continuous},
-    weights=true,
-    descr="Microsoft LightGBM FFI wrapper: Regressor",
+    weights=true
+    # descr="Microsoft LightGBM FFI wrapper: Regressor",
 )
 
+"""
+`LightGBMRegressor`: LightGBM, short for light gradient-boosting machine, is a framework for gradient boosting
```

Comment:
@josephsdavid I don't see a proper header for the docstring, or the code that auto-generates one? Is this because we are adding to an existing docstring? Or maybe some other reason?

Reply:
Resolved.